imports/plugins/core/accounts/client/templates/profile/profile.js

```diff
@@ -26,6 +29,24 @@ Template.accountProfile.onCreated(() => {
  */
 Template.accountProfile.helpers({
 
+  UpdateEmail() {
```

Comment:
In both of these, the data fetching logic should be in a container, not the Blaze template.